src/WebJobs.Script/Description/Workers/WorkerFunctionDescriptorProvider.cs

```diff
@@ -90,5 +102,37 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
 
             return parameters;
         }
+
+        private bool BindingAttributeContainsExpression(IEnumerable<FunctionBinding> bindings)
+        {
+            if (bindings.Count() <= 0)
+            {
+                return false;
+            }
+
+            var bindingExpressionRegex = new Regex(@"{(.*?)\}");
```

Comment:
Rather than new this up each time, we should declare it as a field, and compile it, e.g. https://github.com/Azure/azure-functions-host/blob/852eed9ceef6ef56b431428a8eb31f1bd9c97f3b/src/WebJobs.Script/Description/DotNet/LocalSharedAssemblyProvider.cs#L28